### PR TITLE
Fixed NoMethodError map for NilClass error on `runkeeper.fitness_activities options`

### PR DESCRIPTION
**Pull request by paresharma**
@coop : First, thanks for the awesome work :+1: 

Recently, I ran into this issue when I do something like the following steps:

``` ruby
runkeeper = RunkeeperProfile.get_runkeeper_client '<some token>'
finish=Time.now
start=2.days.ago
options = {start: start, finish: finish}
runkeeper.fitness_activities options
```

I get the following error:

``` ruby
irb(main):007:0* options = {start: start, finish: finish}
=> {:start=>Wed, 11 Mar 2015 01:43:20 CET +01:00, :finish=>2015-03-13 01:43:20 +0100}
irb(main):008:0>
irb(main):009:0* runkeeper.fitness_activities options
NoMethodError: undefined method `map' for nil:NilClass
    from /Users/pareshsharma/Workspace/CRunkeeper/vendor/bundle/gems/run_keeper-0.0.5/lib/run_keeper/activity_request.rb:18:in `request'
    from /Users/pareshsharma/Workspace/CRunkeeper/vendor/bundle/gems/run_keeper-0.0.5/lib/run_keeper/activity_request.rb:34:in `request'
    from /Users/pareshsharma/Workspace/CRunkeeper/vendor/bundle/gems/run_keeper-0.0.5/lib/run_keeper/activity_request.rb:8:in `get_activities'
    from /Users/pareshsharma/Workspace/CRunkeeper/vendor/bundle/gems/run_keeper-0.0.5/lib/run_keeper/request.rb:18:in `fitness_activities'
```

P.S.
Also, I am working on using [Time Scope](http://developer.runkeeper.com/healthgraph/example-api-calls#example-get-retrieving-user-info) while fetching activities from RunKeeper, hopefully I'll create a pull request sometime soon.

### CODE REVIEW

**Comment by donv** [comment]
Hi @coop and @tatey !

Any chance this can be merged?

File: lib/run_keeper/activity_request.rb [hunk 1]
```diff
@@ -15,24 +15,16 @@ def limit_results activities
 
     def request activities = nil
       response   = @runkeeper.request('fitness_activities', @options[:params])
-      activities = response.parsed['items'].map { |activity| Activity.new(activity) }
+      activities = response.parsed['items'] ? response.parsed['items'].map { |activity| Activity.new(activity) } : []
```
Comment (coop) [inline lib/run_keeper/activity_request.rb]: I'd probably have done `Array(response.parsed['items']).map { .... }` instead of a ternary but this will work.

**Comment by coop** [comment]
I'm sorry for not getting to this earlier - I haven't used this gem for quite some time. Glad to know that you're getting use out of it.

**Comment by donv** [comment]
Thanks!

**Pull request merged**